app/Booking.php

```diff
@@ -9,37 +9,52 @@
 class Booking extends Model
 {
     protected $fillable = ['user_id', 'position_id', 'from', 'to'];
+    protected $dates = ['from', 'to'];
 
     public function position(): BelongsTo
     {
         return $this->belongsTo(Position::class);
     }
 
-    public static function canBeMade($position_id, $from, $to)
+    public static function canBeMade($position_id, $from, $to, $excludeID = null)
     {
-        return !self::where('position_id', $position_id)
-            ->where(function ($query) use ($from, $to) {
+        $query = self::where('position_id', $position_id);
```

Comment:
It could be worth only pulling the bookings of a position for the day of the intended booking otherwise this gets intense real quick.